.github/workflows/fullnode-execute-devnet-stable.yaml

```diff
@@ -0,0 +1,57 @@
+# This workflow runs a public fullnode using the `devnet` branch,
+# connects the public fullnode to `devnet` and synchronizes the
+# node using execution syncing to verify that nothing has been broken.
+
+name: "fullnode-execute-devnet-stable"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9/12 * * *"
+  pull_request:
+    paths:
+      - ".github/workflows/fullnode-execute-devnet-stable.yaml"
+
+jobs:
+  fullnode-execute-devnet-stable:
+    timeout-minutes: 300 # Run for at most 5 hours
+    runs-on: high-perf-docker-with-local-ssd
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3
+
+      - uses: ./.github/actions/fullnode-sync
+        with:
+          TIMEOUT_MINUTES: 300 # Run for at most 5 hours
+          BRANCH: devnet
+          NETWORK: devnet
+          BOOTSTRAPPING_MODE: ExecuteTransactionsFromGenesis
+          CONTINUOUS_SYNCING_MODE: ExecuteTransactions
+          DATA_DIR_FILE_PATH: /tmp/
+          NODE_LOG_FILE_PATH: /tmp/node_log
+
+      - name: Upload node logs as an artifact
+        uses: actions/upload-artifact@v3
+        if: ${{ always() }}
+        with:
+          name: node_log
+          path: |
+            /tmp/node_log
+          retention-days: 14
+
+      - name: Post to a Slack channel on failure
+        if: failure()
+        id: slack
+        uses: slackapi/slack-github-action@936158bbe252e9a6062e793ea4609642c966e302 # pin@v1.21.0
+        with:
+          payload: |
+            {
+              "text": "${{ job.status == 'success' && ':white_check_mark:' || ':x:' }} `${{ github.job }}`: <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|link>"
```

Comment:
Simplify this conditional since job.status is always failed?

Reply:
This is copy/pasta from the forge jobs. I'm going to leave it as-is and then once all the tests land, just send out a big PR to update them all 😄 There's also duplication that needs to be cleaned up, too.